sdk/test-utils/test-credential/README.md

```diff
@@ -0,0 +1,38 @@
+# Azure Tools - Test Credential
+
+This library provides a `TokenCredential` implementation for testing Azure SDK packages.
+
+## Getting started
+
+### Installation
+
+Add this library as a dev dependency in your project.
+
+> rush add -p @azure-tools/test-credential --dev
+
+## Key concepts
+
```

Comment:
A two-line code snippet example just for the sake of it would be nice:

```typescript
import { createTestCredential } from "@azure-tools/test-credential";

const credential = createTestCredential();
```